docs/dev/setup.rst

```diff
@@ -371,6 +371,19 @@ you can add a memory limit parameter used in the script in ``package.json`` to t
    # This local change in `package.json` should not be committed.
    --max_old_space_size=5120 # possible higher values are 6144, 7168, and 8192
```

Comment:
This is outdated now, would it be possible to merge the two expanations (from line 366) into one that contains all information?

Reply:
You are right, I addressed the change in [a1353d7](https://github.com/ls1intum/Artemis/pull/4053/commits/a1353d7e33bfb113700a77fc27aa982bdfaf349d)